docs/readme-zephyr.md

```diff
@@ -189,8 +189,8 @@ board thus would involve updating
         label = "image-1";
         reg = <0x00043000 0x37000>;
     };
```

Comment:
the path should be updated too since this is a hwmv1 path that no longer exists

Reply:
The path has been updated.